lxd/container.go

```diff
@@ -1143,6 +1143,24 @@ func instanceLoadAllInternal(dbInstances []db.Instance, s *state.State) ([]Insta
 	return instances, nil
 }
 
+// instanceInstantiate creates the underlying instance type struct and returns it as an Instance.
+func instanceInstantiate(s *state.State, args db.InstanceArgs, cProfiles []api.Profile) (Instance, error) {
```

Comment:
Hmm, this name is going to be a pain to remember and to type right :)

Can we go with something else? `InstanceLoad`?

Reply:
@stgraber yep no prob, that is done.